.gitignore

```diff
@@ -130,5 +130,5 @@ dmypy.json
 .pyre/
 
 # Ignore test files
-*.csv
+# *.csv
```

Comment:
So that we can add new test input/output files. Not sure why it was ignored in the first place. This should be fine as long a we don't check in unnecessary csv files.